models/sap.yml

```diff
@@ -1,11 +1,975 @@
 version: 2
 
 models:
+  - name: sap__0comp_code_attr
+    description: Company codes attributes
```

Comment:
Can you add a bit more context to the description and also a test for this model.

Reply:
Added, brought in mandt to the model.